en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,227 @@
+                        <para>The file name is passed on the <tag
+                                class="attribute">src</tag> attribute.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.template">
+                        <para>The template which is called to import and place
+                            the logo.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.file">
+                        <para>Our file defined at the top of our customization
+                            layer.</para>
+                    </callout>
+                </calloutlist>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>The stylesheet <filename>graphic-pageheader.xsl</filename> creates
+            a logo in the right edge of your page and uses the parameter
+                <parameter>logo.src.path</parameter> to find the logo. By
+            default, a logo is stored in <filename>logo/</filename>. If your
+            logos are in a different location, change it with your XSLT
+            processor. For example, the following line uses the logo(s) image
+            source path <tag>logo.src.path</tag> parameter to find the logo(s).
+            This parameter is used within the position test procedure<xref
+                linkend="co.topic.graphic-pageheader.file"/> to find the right
+            logo.
+            <screen>&lt;xsl:param name="logo.src.path">logo/&lt;/xsl:param>
+            </screen></para>
+        <para/>
+        <para>These points will be submitted later</para>
+        <itemizedlist>
+            <listitem>
+                <para>Logo on recto/verso pages (double-sided docs)</para>
+            </listitem>
+            <listitem>
+                <para>Logo in the footer</para>
+            </listitem>
+            <listitem>
+                <para>Logo and chapter title in the page header</para>
+            </listitem>
+        </itemizedlist>
+    </section>
+    <section role="seealso">
+        <title>See Also</title>
+        <itemizedlist>
+            <listitem xml:id="li.fo.external-graphics">
+                <para>Full parameter description for
+                        <tag>fo:external-graphic</tag><link
+                        xlink:href="http://www.w3.org/TR/xsl/#fo_external-graphic"
+                    /></para>
+            </listitem>
+        </itemizedlist>
+        <para>__________</para>
+        <para>The <emphasis>Opensource</emphasis>-Logo is a trademark which is
+            owned by <link xlink:href="https://opensource.org"/>. This site is
+            not affiliated with or endorsed by the Open Source
+            Initiative.</para>
```

Comment:
Hmn, thinking if this line is better covered with a legalnotice. For the time being, just let it here.